### PR TITLE
fix!: revamp everything & release v2.0.0

### PR DESCRIPTION
**Pull request by myumichi**
* remade themes from scratch
* updated previews
* updated docs
* added fonts used with theme

with these changes i also plan to add multiple different accent colors, later down the line.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,39 +12,75 @@
 </p>
 
 <p align="center">
-  <img src="https://raw.githubusercontent.com/catppuccin/Aliucord/main/assets/preview.png"/>
+  <img src="https://raw.githubusercontent.com/myumichi/aliucord-fork/main/assets/preview.webp"/>
```
Comment (sgoudham) [inline README.md]:
```suggestion
  <img src="./assets/preview.webp"/>
```

File: README.md [hunk 2]
```diff
@@ -14,39 +12,75 @@
...
 </p>
 
+## Previews
+
+<details>
+<summary>🌻 Latte</summary>
+<img src="https://raw.githubusercontent.com/myumichi/aliucord-fork/main/assets/latte-preview.webp"/>
```
Comment (sgoudham) [inline README.md]:
```suggestion
<img src="./assets/latte.webp"/>
```

File: README.md [hunk 3]
```diff
@@ -14,39 +12,75 @@
...
+</details>
+<details>
+<summary>🪴 Frappé</summary>
+<img src="https://raw.githubusercontent.com/myumichi/aliucord-fork/main/assets/frappe-preview.webp"/>
```
Comment (sgoudham) [inline README.md]:
```suggestion
<img src="./assets/frappe.webp"/>
```

File: README.md [hunk 4]
```diff
@@ -14,39 +12,75 @@
...
+</details>
+<details>
+<summary>🌺 Macchiato</summary>
+<img src="https://raw.githubusercontent.com/myumichi/aliucord-fork/main/assets/macchiato-preview.webp"/>
```
Comment (sgoudham) [inline README.md]:
```suggestion
<img src="./assets/macchiato.webp"/>
```

File: README.md [hunk 5]
```diff
@@ -14,39 +12,75 @@
...
+</details>
+<details>
+<summary>🌿 Mocha</summary>
+<img src="https://raw.githubusercontent.com/myumichi/aliucord-fork/main/assets/mocha-preview.webp"/>
```
Comment (sgoudham) [inline README.md]:
```suggestion
<img src="./assets/mocha.webp"/>
```

File: assets/fonts/Manrope-Bold.ttf
Comment (sgoudham) [inline assets/fonts/Manrope-Bold.ttf:1]: Is there a reason why we need to specify these fonts as part of the theme? As a colourscheme, we should focus purely on the highlighting and avoid shipping stylistic changes like fonts.

Let me know if this is included for another reason though.

File: assets/frappe-preview.webp
Comment (sgoudham) [inline assets/frappe-preview.webp:1]: Should be renamed to `frappe.webp`

File: assets/latte-preview.webp
Comment (sgoudham) [inline assets/latte-preview.webp:1]: Should be renamed to `latte.webp`

File: assets/macchiato-preview.webp
Comment (sgoudham) [inline assets/macchiato-preview.webp:1]: Should be renamed to `macchiato.webp`

File: assets/mocha-preview.webp
Comment (sgoudham) [inline assets/mocha-preview.webp:1]: Should be renamed to `mocha.webp`

File: src/frappe.json [hunk 6]
```diff
@@ -1,40 +1,88 @@
...
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Frappe",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin frappé",
+        "version": "1.0.0",
         "license": "MIT",
-        "updater": "https:\/\/raw.githubusercontent.com\/catppuccin\/Aliucord\/main\/src\/frappe.json"
+        "updater": "https:\/\/raw.githubusercontent.com\/myumichi\/aliucord\/main\/src\/frappe.json"
```
Comment (sgoudham) [inline src/frappe.json]:
```suggestion
        "updater": "https:\/\/raw.githubusercontent.com\/catppuccin\/aliucord\/main\/src\/frappe.json"
```

File: src/frappe.json [hunk 7]
```diff
@@ -1,40 +1,88 @@
 {
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Frappe",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin frappé",
+        "version": "1.0.0",
```
Comment (sgoudham) [inline src/frappe.json]: Probably makes more sense to bump this to the next major version?
```suggestion
        "version": "2.0.0",
```

File: src/frappe.json [hunk 8]
```diff
@@ -1,40 +1,88 @@
...
     },
     "background": {},
-    "fonts": {},
+    "fonts": {
+        "whitney_semibold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-Bold.ttf?raw=1",
+        "whitney_bold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-Bold.ttf?raw=1",
+        "ginto_medium": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-ExtraBold.ttf?raw=1",
+        "ginto_bold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-ExtraBold.ttf?raw=1",
+        "whitney_medium": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-SemiBold.ttf?raw=1"
+    },
```
Comment (sgoudham) [inline src/frappe.json]: See comment above for the font files, do we need to ship these?

File: src/latte.json [hunk 9]
```diff
@@ -1,40 +1,87 @@
...
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Latte",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin latte",
+        "version": "1.0.0",
         "license": "MIT",
-        "updater": "https:\/\/raw.githubusercontent.com\/catppuccin\/Aliucord\/main\/src\/latte.json"
+        "updater": "https:\/\/raw.githubusercontent.com\/myumichi\/aliucord\/main\/src\/latte.json"
```
Comment (sgoudham) [inline src/latte.json]:
```suggestion
        "updater": "https:\/\/raw.githubusercontent.com\/catppuccin\/aliucord\/main\/src\/latte.json"
```

File: src/macchiato.json [hunk 10]
```diff
@@ -1,40 +1,88 @@
...
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Macchiato",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin macchiato",
+        "version": "1.0.0",
         "license": "MIT",
-        "updater": "https:\/\/raw.githubusercontent.com\/catppuccin\/Aliucord\/main\/src\/macchiato.json"
+        "updater": "https:\/\/raw.githubusercontent.com\/myumichi\/aliucord\/main\/src\/macchiato.json"
```
Comment (sgoudham) [inline src/macchiato.json]:
```suggestion
        "updater": "https:\/\/raw.githubusercontent.com\/catppuccin\/aliucord\/main\/src\/macchiato.json"
```

File: src/mocha.json [hunk 11]
```diff
@@ -1,40 +1,88 @@
...
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Mocha",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin mocha",
+        "version": "1.0.0",
         "license": "MIT",
-        "updater": "https:\/\/raw.githubusercontent.com\/catppuccin\/Aliucord\/main\/src\/mocha.json"
+        "updater": "https:\/\/raw.githubusercontent.com\/myumichi\/aliucord\/main\/src\/mocha.json"
```
Comment (sgoudham) [inline src/mocha.json]:
```suggestion
        "updater": "https:\/\/raw.githubusercontent.com\/catppuccin\/aliucord\/main\/src\/mocha.json"
```

File: src/latte.json [hunk 12]
```diff
@@ -1,40 +1,87 @@
 {
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Latte",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin latte",
+        "version": "1.0.0",
```
Comment (sgoudham) [inline src/latte.json]: See prior comment about the version

File: src/latte.json [hunk 13]
```diff
@@ -1,40 +1,87 @@
...
     },
     "background": {},
-    "fonts": {},
+    "fonts": {
+        "whitney_semibold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-Bold.ttf?raw=1",
+        "whitney_bold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-Bold.ttf?raw=1",
+        "ginto_medium": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-ExtraBold.ttf?raw=1",
+        "ginto_bold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-ExtraBold.ttf?raw=1",
+        "whitney_medium": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-SemiBold.ttf?raw=1"
+    },
```
Comment (sgoudham) [inline src/latte.json]: See prior comment about the fonts

File: src/macchiato.json [hunk 14]
```diff
@@ -1,40 +1,88 @@
...
     },
     "background": {},
-    "fonts": {},
+    "fonts": {
+        "whitney_semibold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-Bold.ttf?raw=1",
+        "whitney_bold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-Bold.ttf?raw=1",
+        "ginto_medium": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-ExtraBold.ttf?raw=1",
+        "ginto_bold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-ExtraBold.ttf?raw=1",
+        "whitney_medium": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-SemiBold.ttf?raw=1"
+    },
```
Comment (sgoudham) [inline src/macchiato.json]: See prior comment about the fonts

File: src/mocha.json [hunk 15]
```diff
@@ -1,40 +1,88 @@
...
     },
     "background": {},
-    "fonts": {},
+    "fonts": {
+        "whitney_semibold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-Bold.ttf?raw=1",
+        "whitney_bold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-Bold.ttf?raw=1",
+        "ginto_medium": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-ExtraBold.ttf?raw=1",
+        "ginto_bold": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-ExtraBold.ttf?raw=1",
+        "whitney_medium": "https:\/\/github.com\/myumichi\/aliucord\/blob\/main\/assets\/fonts\/Manrope-SemiBold.ttf?raw=1"
+    },
```
Comment (sgoudham) [inline src/mocha.json]: See prior comment about the fonts

File: src/mocha.json [hunk 16]
```diff
@@ -1,40 +1,88 @@
 {
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Mocha",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin mocha",
+        "version": "1.0.0",
```
Comment (sgoudham) [inline src/mocha.json]: See prior comment about version

File: src/macchiato.json [hunk 17]
```diff
@@ -1,40 +1,88 @@
 {
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Macchiato",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin macchiato",
+        "version": "1.0.0",
```
Comment (sgoudham) [inline src/macchiato.json]: See prior comment about version

File: src/frappe.json [hunk 18]
```diff
@@ -1,40 +1,88 @@
 {
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Frappe",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin frappé",
```
Comment (sgoudham) [inline src/frappe.json]: Feels like we should keep the name the same as the existing themes
```suggestion
        "name": "Catppuccin Frappe",
```

File: src/latte.json [hunk 19]
```diff
@@ -1,40 +1,87 @@
 {
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Latte",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin latte",
```
Comment (sgoudham) [inline src/latte.json]:
```suggestion
        "name": "Catppuccin Latte",
```

File: src/macchiato.json [hunk 20]
```diff
@@ -1,40 +1,88 @@
 {
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Macchiato",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin macchiato",
```
Comment (sgoudham) [inline src/macchiato.json]:
```suggestion
        "name": "Catppuccin Macchiato",
```

File: src/mocha.json [hunk 21]
```diff
@@ -1,40 +1,88 @@
 {
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Mocha",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin mocha",
```
Comment (sgoudham) [inline src/mocha.json]:
```suggestion
        "name": "Catppuccin Mocha",
```

**Review by sgoudham — CHANGES_REQUESTED** [review]
A fair few comments on this sorry! 
Just trying to make sure it matches with the existing theme / links are updated.

Also I have some comments surrounding the inclusion of font files, would appreciate if you could help me understand why they were included, thanks!

**Comment by myumichi** [comment]
i can remove the font directory and update the .jsons to remove them if wanted, i figured it _could_ have been optional as they're fairly easy to remove once you open the theme in aliucord

everything else will be updated shortly

**Comment by myumichi** [comment]
the previews reflecting the theme **without the fonts** will take a while updating as everything was done on my phone, sorry in advance

File: README.md [hunk 22]
```diff
@@ -14,39 +12,75 @@
...
+<summary>🪴 Frappé</summary>
+<img src="assets/frappe.webp"/>
+</details>
+<details>
+<summary>🌺 Macchiato</summary>
+<img src="assets/macchiato.webp"/>
+</details>
+<details>
+<summary>🌿 Mocha</summary>
+<img src=assets/mocha.webp"/>
```
Comment (sgoudham) [inline README.md]:
```suggestion
<img src="assets/mocha.webp"/>
```

File: README.md [hunk 23]
```diff
@@ -14,39 +12,75 @@
...
+  - BetterStatusIndicators
+  - BetterChannelIcons
+  - BetterDashless
+  - NoBurnIn
+  - LayoutController
+  - SquareAvatars
 
-## 💝 Thanks to
+- Q: **_Will other accents be added in the future?_**
+- A: Yes, new accents will be added each week. (hopefully)
```
Comment (sgoudham) [inline README.md]: This can be tracked in an issue instead of on the README.

```suggestion
```

File: README.md [hunk 24]
```diff
@@ -14,39 +12,73 @@
...
+  - BetterChannelIcons
+  - BetterDashless
+  - NoBurnIn
+  - LayoutController
+  - SquareAvatars
 
-## 💝 Thanks to
+
+- Q: **_Why can't I see X thing themed?_**
+- A: The introduction of Material You has affected devices on Android 12+, and some areas may not render correctly. If you are on a Custom ROM, please look at [this fix.](https://rentry.co/ThemerFixer)
```
Comment (sgoudham) [inline README.md]:
```suggestion
  A: The introduction of Material You has affected devices on Android 12+, and some areas may not render correctly. If you are on a Custom ROM, please look at [this fix.](https://rentry.co/ThemerFixer)
```

File: README.md [hunk 25]
```diff
@@ -14,39 +12,73 @@
...
+  - BetterChannelIcons
+  - BetterDashless
+  - NoBurnIn
+  - LayoutController
+  - SquareAvatars
 
-## 💝 Thanks to
+
+- Q: **_Why can't I see X thing themed?_**
+  A: The introduction of Material You has affected devices on Android 12+, and some areas may not render correctly. If you are on a Custom ROM, please look at [this fix.](https://rentry.co/ThemerFixer)
```
Comment (sgoudham) [inline README.md]:
```suggestion
- Q: **_Why can't I see X thing themed?_** 
  
  A: The introduction of Material You has affected devices on Android 12+, and some areas may not render correctly. If you are on a Custom ROM, please look at [this fix.](https://rentry.co/ThemerFixer)
```

File: README.md [hunk 26]
```diff
@@ -14,39 +12,74 @@
...
+
+- Q: **_Which plugins are used in the screenshots?_**
+- A: The plugins included are:
+  - BetterChatbox
+  - BetterStatusIndicators
+  - BetterChannelIcons
+  - BetterDashless
+  - NoBurnIn
+  - LayoutController
+  - SquareAvatars
```
Comment (sgoudham) [inline README.md]:
```suggestion
- Q: **_Which plugins are used in the screenshots?_**

  A: The plugins included are:
  	- BetterChatbox
  	- BetterStatusIndicators
  	- BetterChannelIcons
  	- BetterDashless
  	- NoBurnIn
  	- LayoutController
  	- SquareAvatars
```

**Review by sgoudham — APPROVED** [review]
Looks good now, thanks!
The preview images can be updated in a later PR if it'll take some time for you to do it, no rush! 

Happy to merge the PR as is.

**Pull request merged**